applications/RomApplication/tests/hrom_training_utility_test_files/couette_flow_test.mdpa

```diff
@@ -0,0 +1,1463 @@
+Begin ModelPartData
```

Comment:
Why arent we refering to the original file, therefore avoiding copying it in the RomApp folder? I mean:
"input_filename" : "../../../../FluidDynamicsApplication/tests/CouetteFlowTest/couette_flow_test"

Reply:
Change done.